### PR TITLE
Added Azure DevOps CI/CD definition

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
1. Added azure pipelines
2. Added badge
3. Added languages to cucumber (en, ru).
4. Removed travis and Jenkins files.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -9,10 +9,10 @@ gem "json", "1.8.6"
 gem 'rspec', '~> 3.0'
 gem "xamarin-test-cloud", "~> 2.0"
 gem 'retriable', '~> 2.0'
-
+# >=
 # Workflow and build tools
 gem 'rake', '~> 10.3'
-gem 'bundler', '~> 1.6'
+gem 'bundler', '>= 1.6'
```
Comment (maxim-lobanov) [inline Gemfile]: We should not do that.
Why we need this change?

File: Permissions/Images.xcassets/AppIcon.appiconset/Contents.json [hunk 2]
```diff
@@ -97,6 +97,11 @@
       "idiom" : "ipad",
       "filename" : "ipad-pro-icon.png",
       "scale" : "2x"
+    },
+    {
+      "idiom" : "ios-marketing",
+      "size" : "1024x1024",
+      "scale" : "1x"
```
Comment (maxim-lobanov) [inline Permissions/Images.xcassets/AppIcon.appiconset/Contents.json]: I see a lot of noise changes:
```
Permissions/Images.xcassets/**
Permissions_UITest/Permission_demo/.vs/**
 Permissions_UITest/Permission_demo/obj/**
```
Do we really need that? if not - please revert

File: bin/ci/cucumber.rb [hunk 3]
```diff
@@ -2,105 +2,58 @@
 
 require "run_loop"
 require "luffa"
+require "pry"
```
Comment (maxim-lobanov) [inline bin/ci/cucumber.rb]: Please refactor this script:
1) remove noise changes, like spaces, `require "pry"` and etc. 
2) I think we don't need `which_languages` function at all.
You can use something like:
```
languages = ["en", "ru"]
languages.each do |lang|
  args.push("APP_LANG=#{lang}" APP_LOCALE=#{lang}")
```
3) 
```
device = RunLoop::Core.default_simulator(xcode)
device = device.gsub! ' ', ''
device = device.gsub! '(', ''
device = device.gsub! ')', ''
``` 
what we do in this code? Can we refactor it a bit to make more readable?

4)
I see that you determine device two times:
`device` variable,
`match` variable,
Why we need to do that twice?
5) Please remove unused variables

File: Gemfile [hunk 4]
```diff
@@ -9,10 +9,10 @@ gem "json", "1.8.6"
 gem 'rspec', '~> 3.0'
 gem "xamarin-test-cloud", "~> 2.0"
 gem 'retriable', '~> 2.0'
-
+# >=
```
Comment (ivan-nosar) [inline Gemfile]: Please delete it

File: bin/ci/cucumber.rb [hunk 5]
```diff
@@ -8,99 +8,35 @@
...
-  failed_sims = []
-  devices.each do |key, name|
+  languagesLiterals.each do |item|
+    language = "APP_LANG=#{item} APP_LOCALE=#{item}"
     args = [
       "bundle", "exec",
       "cucumber", "-p", "default",
-      "-f", "json", "-o", "reports/#{key}.json",
+      "-f", "json", "-o", "reports/#{item}.json",
       "-f", "junit", "-o", "reports/junit",
```
Comment (ivan-nosar) [inline bin/ci/cucumber.rb]: Test results for Russian locale will overwrite the English ones.
I propose to do something like that:
```
"-f", "junit", "-o", "reports/junit/#{key}",
```

File: azure-pipelines.yml [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    displayName: "Make ipa"
+
+  - script: bundle exec bin/ci/cucumber.rb
+    displayName: "exec cucumber"
+
+  - task: PublishTestResults@2
+    displayName: 'Publish Test Results'
+    inputs:
+      failTaskOnFailedTests: true
+      testResultsFiles: 'reports/junit/*.xml'
```
Comment (ivan-nosar) [inline azure-pipelines.yml]: `testResultsFiles: 'reports/junit/**/*.xml'`

File: features/steps/shared_steps.rb [hunk 7]
```diff
@@ -34,7 +34,7 @@ def tap_row(id)
 
       scroll_to_row_with_mark(id, options)
       wait_for_animations
-
+
```
Comment (ivan-nosar) [inline features/steps/shared_steps.rb]: Please delete it

File: features/steps/shared_steps.rb [hunk 8]
```diff
@@ -369,14 +342,12 @@ def wait_for_alert_dismissed_text
...
-    wait_for_animations
-    sleep(0.4)
-    touch("* marked:'OK'")
-    wait_for_animations
-    wait_for_no_alert
-    wait_for_alert_dismissed_text
-  end
+  wait_for_animations
+  sleep(0.4)
+  touch("* marked:'OK'")
```
Comment (ivan-nosar) [inline features/steps/shared_steps.rb]: Maybe `device_agent.touch()` ?

File: Gemfile [hunk 9]
```diff
@@ -12,7 +12,7 @@ gem 'retriable', '~> 2.0'
 
 # Workflow and build tools
 gem 'rake', '~> 10.3'
-gem 'bundler', '~> 1.6'
+gem 'bundler', '~> 2.0'
```
Comment (jmoody) [inline Gemfile]: This needs to be bundler "1.17.3".  Bundler 2.0 is not allowed on Test Cloud.

Comment (ivan-nosar) [reply Gemfile]: Done

**Pull request merged**